pages/api/auth/[id]/generate.tsx

```diff
@@ -0,0 +1,31 @@
+import { StatusCodes } from 'http-status-codes';
+import type { NextApiHandler, NextApiRequest, NextApiResponse } from 'next';
+import { createVerifyCode } from '../../../../lib/gateways/applications-api';
+
```

Comment:
This file name feels a touch inconsistent - since it's executing an API endpoint called `createVerifyCode`